app/controllers/TaxYearsController.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import controllers.predicates.AuthorisedAction
+import models.TaxYearsResponseModel
+import models.mongo.TaxYearsModel
+
+import javax.inject.Inject
+import play.api.i18n.Lang.logger
+import play.api.libs.json.Json
+import play.api.mvc.{Action, AnyContent, ControllerComponents}
+import services.GetTaxYearsDataService
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import scala.concurrent.ExecutionContext
+
+class TaxYearsController @Inject()(getTaxYearsDataService: GetTaxYearsDataService,
+                                   cc: ControllerComponents,
+                                   authorisedAction: AuthorisedAction)
+                                           (implicit ec: ExecutionContext) extends BackendController(cc) {
+
+  def getTaxYears(nino: String): Action[AnyContent] = authorisedAction.async { implicit user =>
+    getTaxYearsDataService.getTaxYears(nino).map {
+      case Right(taxYearsModel: TaxYearsModel) =>
+        logger.debug(s"[TaxYearsController][getTaxYears] - Successful Response: $taxYearsModel")
+        Ok(Json.toJson(TaxYearsResponseModel(taxYearsModel.taxYears)))
```

Comment:
the frontend usage of the API is expecting them in a specific order so could you add .sorted to the taxYears.

Reply:
The taxYears is already sorted internally.
https://github.com/hmrc/income-tax-calculation/pull/45/files#diff-359d5439cfa0e7183b08c6bf8d1e0be6d3c6b9319b868aa9b980720b748c8be9R38
https://github.com/hmrc/income-tax-calculation/pull/45/files#diff-359d5439cfa0e7183b08c6bf8d1e0be6d3c6b9319b868aa9b980720b748c8be9R38